de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/grid/cells/DiskResourceDotMenuDefaultAppearance.java

```diff
@@ -0,0 +1,112 @@
+package org.iplantc.de.theme.base.client.diskResource.grid.cells;
+
+import org.iplantc.de.client.models.diskResources.DiskResource;
+import org.iplantc.de.diskResource.client.views.grid.cells.DiskResourceDotMenuCell;
+import org.iplantc.de.resources.client.IplantResources;
+import org.iplantc.de.resources.client.messages.IplantDisplayStrings;
+import org.iplantc.de.theme.base.client.apps.cells.AppDotMenuDefaultAppearance;
+import org.iplantc.de.theme.base.client.diskResource.DiskResourceMessages;
+
+import com.google.gwt.cell.client.Cell;
+import com.google.gwt.core.client.GWT;
+import com.google.gwt.resources.client.ImageResource;
+import com.google.gwt.safehtml.shared.SafeHtmlBuilder;
+
+public class DiskResourceDotMenuDefaultAppearance implements DiskResourceDotMenuCell.DiskResourceDotMenuAppearance {
+
+    private IplantResources iplantResources;
+    private IplantDisplayStrings iplantDisplayStrings;
+    private DiskResourceMessages diskResourceMessages;
+    private AppDotMenuDefaultAppearance appDotMenuDefaultAppearance;
+    private DiskResourceDotMenuDisplayStrings displayStrings;
+
+    public DiskResourceDotMenuDefaultAppearance() {
+        this(GWT.<IplantResources>create(IplantResources.class),
+             GWT.<IplantDisplayStrings>create(IplantDisplayStrings.class),
+             GWT.<DiskResourceMessages>create(DiskResourceMessages.class),
+             GWT.<AppDotMenuDefaultAppearance>create(AppDotMenuDefaultAppearance.class),
+             GWT.<DiskResourceDotMenuDisplayStrings>create(DiskResourceDotMenuDisplayStrings.class));
+    }
+
+    public DiskResourceDotMenuDefaultAppearance(IplantResources iplantResources,
+                                                IplantDisplayStrings iplantDisplayStrings,
+                                                DiskResourceMessages diskResourceMessages,
+                                                AppDotMenuDefaultAppearance appDotMenuDefaultAppearance,
+                                                DiskResourceDotMenuDisplayStrings displayStrings) {
+
+        this.iplantResources = iplantResources;
+        this.iplantDisplayStrings = iplantDisplayStrings;
+        this.diskResourceMessages = diskResourceMessages;
+        this.appDotMenuDefaultAppearance = appDotMenuDefaultAppearance;
+        this.displayStrings = displayStrings;
+    }
+
+    @Override
+
+    public void render(Cell.Context context, DiskResource value, SafeHtmlBuilder sb, String debugId) {
+        appDotMenuDefaultAppearance.render(context, sb, debugId);
+    }
+
+    @Override
+    public ImageResource dotMenuIcon() {
+        return appDotMenuDefaultAppearance.dotMenuIcon();
+    }
+
+    @Override
+    public String favoriteText(DiskResource diskResource) {
```

Comment:
(semi-random spot, I mostly am reviewing from the screenshot)

I think this string has a period at the end, where everything else in the menu doesn't. It looks a bit weird that way, not sure the ideal way to fix (I personally like the ones without periods better)

Reply:
I like them better without the periods too, I'll update 👍